.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,25 @@
+name: scheduled
+on:
+  schedule:
+    # Runs at 00:00 UTC every Sunday, Tuesday, Thursday.
+    - cron: '0 0 * * 0,2,4'
+
+jobs:
+  trigger-workflow:
+    strategy:
+      matrix:
+        branch: ["main", "release-1.3"]
+        wf_id: ["validate", "test"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger ${{ matrix.wf_id }} workflow on ${{ matrix.branch}} branch
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Can we explicitly set `permissions:` for the token?

Reply:
As far as I understand, permissions for this token are set in https://github.com/opencontainers/runc/settings/actions, under "Workflow permissions".

Also, the file itself sets permissions to create actions and read contents.

Reply:
Would this work if we set there: "Read repository contents and packages permissions" and not set "Allow GitHub Actions to create and approve pull requests"?